lib/setup.js

```diff
@@ -23,13 +23,14 @@ bluebird.config({
  * @returns {Promise}
  */
 module.exports = function (options) {
-  let app, engines, providers, router, sessionStore;
+  let app, engines, providers, router, sessionStore, searchConfig;
 
   options = options || {};
   app = options.app || express(); // use new express app if not passed in
   engines = options.engines; // will be undefined if not passed in
   providers = options.providers || [];
   sessionStore = options.sessionStore;
+  searchConfig = options.search || {}; // Config object to pass to the search setup
```

Comment:
Because that's the default we'll want to pass to the search service for now. We'll eventually make it so if it doesn't exist the search service doesn't instantiate